Backpack/Calendar/Classes/BPKCalendarTrafficLightCellData.m

```diff
@@ -0,0 +1,96 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKCalendarTrafficLightCellData.h"
+
+#import <Backpack/Common.h>
+#import <Backpack/Color.h>
+
+@implementation BPKCalendarTrafficLightCellData
+
++ (BPKCalendarTrafficLightCellData *)normal {
+    static dispatch_once_t onceToken;
+    static BPKCalendarTrafficLightCellData  *cellData;
+
+    dispatch_once(&onceToken, ^{
+        BPKAssertMainThread();
```

Comment:
I don't think we want this here. In general the code here should be thread safe and doesn't have to be on the main thread, AFAIK `UIColor` are okay to use from non-main thread

Reply:
Tru dat!